docs/modules/ROOT/pages/getting-started/assembly-configuring-the-registry.adoc

```diff
@@ -4,11 +4,13 @@ include::{mod-loc}shared/all-attributes.adoc[]
 [id="configuring-the-registry"]
 = Configuring {registry} on OpenShift
 
-This chapter explains how to configure optional settings for {registry} health checks on OpenShift:
+This chapter explain various optional configuration settings for {registry}, such as health checks settings on OpenShift or Logging settings.
```

Comment:
Suggest: 
This chapter explains various optional configuration settings for {registry}, such as health checks on OpenShift and logging.